cmd/cloud/server_flag.go

```diff
@@ -0,0 +1,213 @@
+package main
+
+import (
+	"time"
+
+	toolsAWS "github.com/mattermost/mattermost-cloud/internal/tools/aws"
+	"github.com/spf13/cobra"
+)
+
+type supervisorOption struct {
```

Comment:
nit: `supervisorOptions` is probably a bit better along with the other options, but not a big deal. 0/5